src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,96 @@
 package juja.sqlcmd;
 
+import java.sql.*;
+
 public class Application {
+
+
+    private final static String SERVER = "localhost";
+    private final static String PORT = "5432";
+    private final static String DB_NAME = "sqlcmd";
+    private final static String USER_NAME = "sqlcmd";
+    private final static String PASSWORD = "sqlcmd";
+    private Connection connection;
+
     public static void main(String[] args) {
+        try {
+            new Application().simpleSQL();
+        } catch (ClassNotFoundException | SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void simpleSQL() throws ClassNotFoundException, SQLException {
+        connection = getConnection();
+        System.out.println(getTablesName());
+        createTable("user");
+        System.out.println(getTablesName());
+
+
+
+    }
+
+    private void createTable(String tableName) throws SQLException {
+        if (isCorrectTableName(tableName)) {
+            System.out.println("Wrong table name.");
+            return;
```

Comment:
Sometimes i see several return considered a bad practice. Wrap all validation in particular method.
For example 
if (allParamatersIsCorrect()) {
...//do business logic
}

Reply:
Thanks for the advice. 
I'll fix it